teamB/simchaewon/app.js

```diff
@@ -0,0 +1,33 @@
+const http=require("http");
+const express=require("express");
+const cors=require("cors");
+const morgan=require("morgan");
+
+require("dotenv").config();
```

Comment:
해당 부분은 최상단으로 올려서 작성해주세요 !
```suggestion
require("dotenv").config();
```